packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -1779,6 +1780,161 @@ void main() {
       lessThan(600 - windowPaddingBottom), // Device height is 600.
     );
   });
+
+  group('feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    testWidgets('PopupMenuButton with enabled feedback', (WidgetTester tester) async {
+      const bool enableFeedback = true;
+
+      await tester.pumpWidget(
```

Comment:
A simple pattern for tests like this one and the ones that follow is to define a function that generates a frame with the configuration to be tested. In this case something like: 

```dart
Widget buildFrame({ bool? widgetEnableFeedack, bool? themeEnableFeedback }) {
   ...
}
```

And then, in just one test, check all four combinations. Like
```
await buildFrame(widgetEnableFeedback: true);
await tester.tap(find.byType(PopupMenuButton));
await tester.pumpAndSettle();
// check the feedback
// dismiss the menu by tapping on an item and another pumpAndSettle()

// test the next combination of widgetEnableFeedback, themeEnableFeedback values
```